flink/v1.12/flink/src/main/java/com/netease/arctic/flink/read/source/ArcticScanContext.java

```diff
@@ -49,6 +51,8 @@ public class ArcticScanContext extends ScanContext implements Serializable {
   protected ArcticScanContext(
       boolean caseSensitive,
       Long snapshotId,
+      StreamingStartingStrategy startingStrategy,
+      Long startSnapshotTimestamp,
```

Comment:
what's the different with `asOfTimestamp`?

Reply:
asOfTimestamp is used for BATCH scan mode, and startSnapshotTimestamp is used for INCREMENTAL_APPEND scan mode.